changelog/12534.bugfix.md

```diff
@@ -0,0 +1,2 @@
+Buttons in `rasa shell` are now correctly parsed.
```

Comment:
```suggestion
Rich responses containing buttons with parentheses characters are now correctly parsed.
```
Added a bit more detail, and also I don't think this is just a rasa shell utility, please double-check and reintroduce if it's just rasa shell.

Reply:
Yup it's not just shell it's interactive too.